danielssonn/ethTO
# fix: fix eslint

## Purpose

1. Installed `eslint-config-prettier` in `client` and updated the ESLint config
2. Moved the root `.prettierignore` and `.prettierrc` files into `client` as I noticed that the `protocol` directory has its own Prettier-related files

## Testing Instructions

1. Confirm that ESLint behaves as expected (optional):

- Pull down the branch
- Move into the `client` directory
- Make a change and run `npm run lint`
- Confirm that there are no ESLint problems

3. Review code

## Checklist
